From f0c622e431d54793f73d72abfc733a6f6d723035 Mon Sep 17 00:00:00 2001
From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Wed, 2 Nov 2011 00:14:37 +0100
Subject: [PATCH 2/9] don't regen docs if perl is not found

If we're using a cvs snapshot which updates the source files, and
perl isn't installed yet, then we can't regen the docs.  Not a big
deal, so just whine a little and continue on our merry way.

This patch was taken from gentoo.
http://bugs.gentoo.org/60132

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 manual/Makefile | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/manual/Makefile b/manual/Makefile
index f2f694f..688d9e2 100644
--- a/manual/Makefile
+++ b/manual/Makefile
@@ -104,9 +104,14 @@ $(objpfx)dir-add.texi: xtract-typefun.awk $(texis-path)
 $(objpfx)libm-err.texi: $(objpfx)stamp-libm-err
 $(objpfx)stamp-libm-err: libm-err-tab.pl $(wildcard $(foreach dir,$(sysdirs),\
 						     $(dir)/libm-test-ulps))
+ifneq ($(PERL),no)
 	pwd=`pwd`; \
 	$(PERL) $< $$pwd/.. > $(objpfx)libm-err-tmp
 	$(move-if-change) $(objpfx)libm-err-tmp $(objpfx)libm-err.texi
+else
+	echo "Unable to rebuild math docs, no perl installed"
+	touch libm-err.texi
+endif
 	touch $@
 
 # Package version and bug reporting URL.
-- 
2.6.4.442.g545299f

